framework/contracts/account/manager/src/contract.rs

```diff
@@ -206,6 +206,16 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> M
                         cw_ownable::Action::AcceptOwnership => update_governance(deps.storage)?,
                         _ => vec![],
                     };
+                    let mut info = info;
+                    // if new governance is sub-account trick cw-ownable to think it was called by proxy
```

Comment:
I am very confused by this, could you elaborate on why this is needed?

Reply:
Ahhhh I get it!! Okey haha, sorry

Reply:
> I am very confused by this, could you elaborate on why this is needed?

Our `execute_update_ownership` macro checks new owner against `info.sender`, and to support top-level admin claims we need to edit sender when it's called by the top-level admin :)

Reply:
moved it into `update_governance` function as well, so pending owner can't renounce himself(which is okay but still minor bug)